docs/book/installation.rst

```diff
@@ -96,6 +96,19 @@ Accessing the Administration Panel
     Have a look at the ``/admin`` url, where you will find the administration panel.
     Remember that you have to be logged in as an administrator using the credentials provided while installing Sylius.
 
+How to start developing? - Project Structure
+--------------------------------------------
+
+After you have successfully gone through the installation process of **Sylius-Standard** you are probably going to start developing within the framework of Sylius.
+
+In the root directory of your project you will find these important subdirectories:
+
+* ``app/config/`` - here you will be adding the yaml configuration files including routing, security, state machines configurations etc.
```

Comment:
Can we just add a link to symfony structure description? And write something like: "As was written before Sylius is Symfony based project and brings same folder structure as a base framework"